src/Neo/Wallets/Wallet.cs

```diff
@@ -137,7 +138,7 @@ protected Wallet(string path, ProtocolSettings settings)
         public WalletAccount CreateAccount()
         {
             byte[] privateKey = new byte[32];
-        generate:
+generate:
```

Comment:
I think that this tabulation is wrong

Reply:
This is the unique issue that I found reviewing the 500 files 😪

Reply:
`trim_trailing_whitespace = true`

Reply:
I think that is a problem with the goto, maybe we have only this one

Reply:
@shargon, when you think something should be changed, may you please just add suggestion and update it? I am not familer with editconfig, so i will take whatever your suggestions.